index.bs

```diff
@@ -440,6 +462,11 @@ dictionary IntersectionObserverEntryInit {
 		to intersecting with a zero-area intersection rect (as will happen with
 		edge-adjacent intersections, or when the {{IntersectionObserverEntry/boundingClientRect}}
 		has zero area).
+	: <dfn>isVisible</dfn>
+	::
+		True if {{IntersectionObserver/trackVisibility}} is <code>true</code>
+		and the <a>visibility</a> algorithm,
```

Comment:
For the purpose of feature detection, how about setting `isVisible` to `null` if `trackVisibility` is false? I think that's also more consistent with other IDL I have seen.

Reply:
you can't set a boolean to null, can you? Honest question -- I'm not sure what the WebIDL can express, nor what would be most consistent with similar APIs. This feels kind of off, but a permanent `false` when tracking isn't enabled wouldn't be terrible. The `true` specified in the algorithm below will lead to trouble.

Any thoughts, @annevk or @zcorpan?

Reply:
I'm not an IDL expert, but for the record chromium's IDL code generator allows null-able booleans. I wrote a chromium patch which would implement the behavior I proposed, and it works.

Reply:
You can have nullable booleans (the IDL of this PR does not do that btw), but generally it would be better to use an enum.

Reply:
I'm not sure an enum will work here, because we need two different values that will evaluate to `false` when cast to boolean.

If there are no objections, I'll just make it a permanent `false` when tracking isn't enabled. I think that's probably what we want from a developer ergonomics no-surprises point of view. Feature detection can always be done via `IntersectionObserverEntry.prototype.hasOwnProperty('isVisible')`.